codes/codes.go

```diff
@@ -215,6 +216,12 @@ var strToCode = map[string]Code{
 	`"UNAUTHENTICATED"`:     Unauthenticated,
 }
 
+// FromString is used to get code from string
+func FromString(s string) (Code, bool) {
```

Comment:
This doesn't reverse `Code.String()`, which could cause confusion.  Maybe it just needs a different name, like `FromJSONString`?  Or `FromEnumName`?  We could also potentially change `Code.String()`, but I'm not sure about the backward compatibility requirements of `String` methods.

Reply:
I agree with you.
I prefer to write like "DEADLINE_EXCEEDED" instead of "DeadlineExceeded" or "DEADLINE EXCEEDED" because the enum in https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto is defined in this way.
Also, it looks good to do this conversion from the `Code_value` of `google.golang.org/genproto/googleapis/rpc/code` instead of reading `strToCode`, although `Code_value` is not a functional API.
The impact of changing `Code.String()` may be huge, although I think it is more reasonable to let it follow the definition of `enum`. There shouldn’t be so many ways to express one thing.